src/commands/jeopardy.js

```diff
@@ -0,0 +1,127 @@
+const Discord = require("discord.js");
+const errors = require("../utils/errors.js");
+const config = require("../botconfig");
+const {get} = require('snekfetch');
+
+var liveRound = 0;
+var host;
+var jeopardyChannel;
+var players = [];
+var scores = [];
+var playerIds = [];
+
+module.exports.run = async (bot, message, args) => {
+    var argument = args.toString().replace(/"/gm,""); //causes a lot of errors but necessary for arg parsing
+    if(host == null && jeopardyChannel == null){
+        host = message.author;
+        jeopardyChannel = message.channel;
+    }
+    if(argument == "" && message.channel == jeopardyChannel){ //Good for now
+        if(liveRound == 0){
+            liveRound = 1;
+            players = [];
+            scores = [];
+            playerIds = [];
+            message.channel.send(`Welcome to Jeopardy, hosted by ${message.author}! View rules with ?jeopardy "rules".`);
+        }
+        else if(liveRound == 1){
+            message.channel.send(`There is already a live round of Jeopardy.`);
+        }
+    }
+    if(argument == "next" && message.channel == jeopardyChannel){ //TODO: Timer
+        if(liveRound == 0){
+            message.channel.send(`There isn't a live round of Jeopardy. Start one with ?jeopardy.`);
+        }
+        if(liveRound == 1 && message.author == host){
+            let { answer, question, value, category } = await getQuestion();
+            answer = answer.replace(/<(?:.|\n)*?>/gm, '');
+            var answered = 0;
+            console.log(answer); //for bugtesting, remove before merging to master
```

Comment:
remove this console.log

Reply:
the console.log is now an intended feature